src/server/api/common_api.py

```diff
@@ -124,11 +127,12 @@ def get_animals(matching_id):
             shelterluv_id = row["source_id"]
             person_url = f"http://shelterluv.com/api/v1/people/{shelterluv_id}"
             person_details = requests.get(person_url, headers={"x-api-key": SHELTERLUV_SECRET_TOKEN}).json()
+            result["person_details"]["shelterluv_short_id"] = person_details["ID"]
```

Comment:
can use `_.set(result, "person_details.shelterluv_short_id", person_details["ID"])` for the same resons mentioned above